### ISSUE TITLE
Release supporting NodeJS 10?

### ISSUE THREAD

**Issue by csantanapr** [ibmdb/node-ibm_db#482]
I need to support NodeJS10 runtime with the ibm DB npm package included for IBM Cloud Functions.
I see that this commit added the support https://github.com/ibmdb/node-ibm_db/commit/5457060a576251c649003a5b21c14f5e26d64f90

When is the  new release expected to be publish to npm registry?

**Comment by rhtpandeyIN** [comment]
Hi @csantanapr ,

Next release date is 7th Dec 2018. Thanks.

Closing the issue as it's not a bug, but I will update the progress and information in this closed issue for the tracking purpose. You can still reopen it if required.

**Comment by csantanapr** [comment]
Please re-open this issue for tracking purposes, and also to avoid other people opening a duplicate issue. Everyone waiting on nodejs10 can use this issue to track progress, once release the issue can be close and this will notified everyone subscribe.

I can't open it since I don't have write access on this issue

**Comment by huineng** [comment]
let's add that node 11.x is support on (my) mac and on docker (from node 11.1) by installing it directly from "npm install https://github.com/ibmdb/node-ibm_db"

**Comment by bimalkjha** [comment]
@huineng thanks for the confirmation.

**Comment by csantanapr** [comment]
@rhtpandeyIN are we still on track for a release on the 7th?

**Comment by rhtpandeyIN** [comment]
Yes @csantanapr we are :)

**Comment by bimalkjha** [comment]
ibm_db@2.5.0 supports node.js v10.x & v11.x. thanks.

**Comment by alenaSchmickl** [comment]
Hi, I used the most recent package available, but now get a mismatch as the node_module_version is 67 (for node v11), however I am using node version v10 which implies a node_module_version 64.  This results in  a version mismatch error. Will this issue be fixed once the package is available via npm?

**Comment by Owenzh** [comment]
nodejs: v10.19.0
npm: v6.13.4
ibm_db: v2.6.4

Can build the app, but start app and visit data list page show error:

node_modules\ibm_db\build\Release\odbc_bindings.node :: "Error: The specified module could not be found.  at Object.Module._extensions..node (internal/modules/cjs/loader.js:807:18)


I can find this odbc_bindings.node file.
Can anyone help me out?

Thanks.

**Comment by bimalkjha** [comment]
@Owenzh Its a closed issue as ibm_db supports node.js v10.x since long back. Please open a new issue with more details like what is your platform and how to reproduce the problem. Thanks.
